sw/device/silicon_creator/manuf/keys/fake/fake_ca.conf

```diff
@@ -15,5 +15,5 @@ CN=Google Engineering ICA
 [v3_ca]
 subjectKeyIdentifier = hash
 authorityKeyIdentifier = keyid:always,issuer:always
-basicConstraints = critical,CA:true,pathlen:0
+basicConstraints = critical,CA:true,pathlen:3
```

Comment:
pathlen of 4 to enable CDI_1 to endorse another cert in test flows?

Reply:
https://stackoverflow.com/questions/6616470/certificates-basic-constraints-path-length

Perhaps we should just omit the pathlen constraint from the fake CA.

Reply:
I eliminated the `pathlen` constraint.  We should still seek guidance on this when generating the real endorsement certificate to be used in production.